lec09-Randomization-tests.Rmd

```diff
@@ -144,10 +146,16 @@ df_females <- data.frame(
 # Let's look at the male and female data
 ggplot() +
     ylab("Count") + xlab("Body width") +
-    geom_histogram(data = df_males, aes(x = width), bins = 30, fill = "red", alpha = 0.4, colour = "black") +
-    geom_histogram(data = df_females, aes(x = width), bins = 30, fill = "blue", alpha = 0.4, colour = "black") +
-    geom_vline(data = df_males, aes(xintercept = mean(width)), size = 1, linetype = "dashed", colour = "red") +
-    geom_vline(data = df_females, aes(xintercept = mean(width)), size = 1, linetype = "dashed", colour = "blue") +
+    geom_histogram(data = df_males, aes(x = width), 
+                   bins = 30, fill = "red", alpha = 0.4, 
+                   colour = "black") +
+    geom_histogram(data = df_females, aes(x = width), 
+                   bins = 30, fill = "blue", alpha = 0.4, 
+                   colour = "black") +
+    geom_vline(data = df_males, aes(xintercept = mean(width)), 
+               size = 1, linetype = "dashed", colour = "red") +
+    geom_vline(data = df_females, aes(xintercept = mean(width)), 
+               size = 1, linetype = "dashed", colour = "blue") +
```

Comment:
You know, instead of having two datasets to use to make two plots, you can simplify the code substantially by using one dataset with both male and female. Then using `colour` aes or `group` or even `facet_grid`.  That way you have way less code to deal with. For instance:

```
ggplot(df, aes(x = width, fill = Sex, colour = Sex) +
  geom_histogram(bins = 30, colour = "black", alpha = 0.4) +
  geom_vline(aes(xintercept = mean(width)), size = 1, linetype = "dashed")
```